### PR TITLE
Feature/change alg

### PR DESCRIPTION
**Pull request by Verntrax**

### CODE REVIEW

File: src/experiment.R [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  experiment(model_gen = function(training_data) {
+    return(anomaly_detector(algorithm, training_data, clusters_count))
+  }, extract_normal_anomaly)
+}
+
+classification_experiment = function(extract_normal_anomaly, nu, gamma) {
+  experiment(model_gen = function(training_data) {
+    return(svm(training_data, y=!vector(length=nrow(training_data)), type="one-classification", nu=nu, gamma=gamma))
+  }, extract_normal_anomaly)
+}
```
Comment (unitatem) [inline src/experiment.R]: \n

File: src/grouping_wrapper.R [hunk 2]
```diff
@@ -1,74 +1,61 @@
...
+  qnt = quantile(vec, probs=c(.25, .75))
+  H = 1.5 * IQR(vec)
+  vec[vec < (qnt[1] - H)] <- NA
+  vec[vec > (qnt[2] + H)] <- NA
+  
+  return(vec)
+}
+
+anomaly_detector = function(algorithm, data, clusters) {
+  border = vector("numeric", length=clusters)
```
Comment (unitatem) [inline src/grouping_wrapper.R]: move to line 33 (before for loop)

File: src/grouping_wrapper.R [hunk 3]
```diff
@@ -1,74 +1,61 @@
...
+  max = apply(data, 2, max)
+  min = apply(data, 2, min)
+  data = normalize(data, max, min)
+  
+  clustering = algorithm$get_cluster_params(data, clusters)
+  
+
```
Comment (unitatem) [inline src/grouping_wrapper.R]: rm blank line

File: src/main_svm.R [hunk 4]
```diff
@@ -0,0 +1,8 @@
+rm(list = ls())
+
+source("src/config.R")
+source("src/experiment.R")
+
+message("START")
+classification_experiment(extract_normal_anomaly, nu, gamma)
+message("END")
```
Comment (unitatem) [inline src/main_svm.R]: \n

**Review by unitatem — CHANGES_REQUESTED** [review]

File: src/experiment.R (hunk 1 above)
Comment (Verntrax) [reply src/experiment.R]: Done

File: src/grouping_wrapper.R (hunk 2 above)
Comment (Verntrax) [reply src/grouping_wrapper.R]: Done

File: src/grouping_wrapper.R (hunk 3 above)
Comment (Verntrax) [reply src/grouping_wrapper.R]: Done

File: src/main_svm.R (hunk 4 above)
Comment (Verntrax) [reply src/main_svm.R]: Done

**Review by unitatem — APPROVED** [review]

**Pull request merged**